src/page/ProductAddedModal.page.ts

```diff
@@ -1,11 +1,12 @@
-import { element, by, ElementFinder, promise } from 'protractor';
+import { element, by, ElementFinder, ExpectedConditions, browser } from 'protractor';
 
 export class ProductAddedModalPage {
   private get proceedToCheckoutButton(): ElementFinder {
     return element(by.css('[style*="display: block;"] .button-container > a'));
   }
 
-  public checkoutProduct(): promise.Promise<void> {
-    return this.proceedToCheckoutButton.click();
+  public async checkoutProduct(): Promise<void> {
+    await browser.wait(ExpectedConditions.elementToBeClickable(this.proceedToCheckoutButton), 5000);
```

Comment:
is this the only Expected condition in all the test?

Reply:
is the only one who fail, after do this the test works fine.

Reply:
OK
LGTM then